model/trace/rpc.yaml

```diff
@@ -55,21 +55,18 @@ groups:
           recommended: If different than `server.port` and if `server.socket.address` is set.
       - ref: network.transport
       - ref: network.type
-      - ref: server.address
+      - ref: server.domain
         requirement_level: required
         brief: >
           RPC server [host name](https://grpc.github.io/grpc/core/md_doc_naming.html).
         note: >
           May contain server IP address, DNS name, or local socket name. When host component is an IP address,
           instrumentations SHOULD NOT do a reverse proxy lookup to obtain DNS name and SHOULD set
-          `server.address` to the IP address provided in the host component.
+          `server.domain` to the IP address provided in the host component.
+      - ref: server.address
       - ref: server.port
         requirement_level:
           conditionally_required: See below
-    constraints:
```

Comment:
I removed this, since this was already meaningless since one of the any_of attributes was already "required".